lms/templates/ux/reference/course-skeleton.html

```diff
@@ -0,0 +1,71 @@
+## Override the default styles_version to the Pattern Library version (version 2)
+<%! main_css = "style-main-v2" %>
+
+<%page expression_filter="h"/>
+<%inherit file="/main.html" />
+
+<%block name="pagetitle">Course Skeleton</%block>
+
+<%block name="bodyclass">pattern-library</%block>
+
+<%block name="content">
+<nav class="wrapper-course-material">
+    <div class="course-material">
+        <ol class="tabs course-tabs">
+            <li class="tab">
+                <a href="index.html" class="">
```

Comment:
Good catch. I just grabbed the HTML from a rendered page, and I guess that's how it spits it out. I've cleaned this up.